analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeExtensions.Roslyn.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.CodeAnalysis.CSharp.Extensions;
+
+[ExcludeFromCodeCoverage]
+internal static class SyntaxNodeExtensions
+{
+    /// <summary>
+    /// Returns the left hand side of a conditional access expression. Returns c in case like a?.b?[0].c?.d.e?.f if d is passed.
+    /// </summary>
+    /// <remarks>Copied from <seealso href="https://github.com/dotnet/roslyn/blob/18f20b489/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs#L188">
```

Comment:
This (and also the methods below) is a bit different syntactically from the `Copy of` comments below.

I think it's fine, since we are going to introduce a dedicated attribute soon, that will make the format uniform.

An alternative would be to add a "normal" comment `Copy of` above `/// <summary>`. It would probably even be more correct since the documentation itself is a copy of the documentation in the Roslyn repository. 

Everything below such `// Copy of` comment would be 1-to-1 with the corresponding code on Roslyn, whereas this is not the case right now, where a diff would reveal added code in the middle of the documentation.